cellarium/ml/models/common/encoder.py

```diff
@@ -0,0 +1,113 @@
+from collections.abc import Iterable
+from typing import Callable, Optional
+
+import torch
+from torch import nn
+from torch.distributions import Normal
+
+from cellarium.ml.models.common.fclayer import FCLayers
+
+
+def _identity(x):
+    return x
+
+
+class Encoder(nn.Module):
```

Comment:
Same comment as earlier about `decoder.py` not being general enough to be under `common`.